shards/gfx/renderer_storage.hpp

```diff
@@ -43,7 +43,7 @@ struct RendererStorage {
     if (textureData.externalView)
       return textureData.externalView;
 
-    assert(textureData.texture);
+    assert(textureData.texture && "Invalid texture");
```

Comment:
`shassert` preferred so will assert in RelWithDebInfo

Reply:
Changed them all to shassert and moved shassert to shared header